mapping/svg-maps.md

```diff
@@ -0,0 +1,15 @@
+# About SVG files
+SVG files, or scalable vector graphic files, are a unique image type that work well for making maps. Maps made with SVG files allow us access to the discrete geographies within the map using JavaScript, which also opens up client-side map rendering. This makes SVG maps a powerful tool when working with front-end frameworks like React.js.
+
```

Comment:
Can you say a bit more about what sorts of tasks SVG maps are and aren't good for?